src/sign-authorization.test.ts

```diff
@@ -0,0 +1,241 @@
+import { bufferToHex, privateToAddress } from '@ethereumjs/util';
+
+import {
+  signAuthorization,
+  recoverAuthorization,
+  Authorization,
+  hashAuthorization,
+} from './sign-authorization';
+
+const testPrivateKey = Buffer.from(
+  '4af1bceebf7f3634ec3cff8a2c38e51178d5d4ce585c52d6043e5e2cc3418bb0',
+  'hex',
+);
+
+const testAddress = bufferToHex(privateToAddress(testPrivateKey));
+
+const testAuthorization: Authorization = [
+  8545,
+  '0x1234567890123456789012345678901234567890',
+  1,
+];
+
+const expectedAuthorizationHash = Buffer.from(
+  'b847dee5b33802280f3279d57574e1eb6bf5d628d7f63049e3cb20bad211056c',
+  'hex',
+);
+
+const expectedSignature =
+  '0xebea1ac12f17a56a514dfecbcbc8bbee7b089fa3fcee31680d1e2c1588f623df7973cab74e12536678995377da38c96c65c52897750b73462c6760ef2737dba41b';
+
+describe('signAuthorization', () => {
+  describe('signAuthorization()', () => {
+    it('should produce the correct signature', () => {
```

Comment:
Minor, `should` is redundant in test names?  `produces the correct signature` for example.

Reply:
You are correct - it could be removed, but `should` is consistent across all of the existing tests 🤷

Reply:
It's a legacy pattern in lots of repositories that we're slowing moving aways from, but not urgent now.